### PR TITLE
Task chat popup

### PR DESCRIPTION
**Pull request by rishi9684**
#added tabs for backstage and live
#added iframe

### CODE REVIEW

File: react/features/chat/components/web/Chat.js [hunk 1]
```diff
@@ -86,6 +89,7 @@ class Chat extends AbstractChat<Props> {
         return (
             <>
                 { this._renderPanelContent() }
+                {/* <Tabs tabs={tabs} /> */}
```
Comment (SukantGujar) [inline react/features/chat/components/web/Chat.js]: Remove

File: react/features/chat/components/web/Chat.js [hunk 2]
```diff
@@ -152,20 +158,43 @@ class Chat extends AbstractChat<Props> {
...
-                    </ChatDialog>
-                );
-            } else {
-                ComponentToRender = (
-                    <>
-                        { this._renderChatHeader() }
-                        { _showNamePrompt ? <DisplayNameForm /> : this._renderChat() }
-                    </>
-                );
+            if(interfaceConfig.CHAT_TOKEN && interfaceConfig.CHAT_TOKEN) {
```
Comment (SukantGujar) [inline react/features/chat/components/web/Chat.js]: Why && between the same values?

File: react/features/chat/components/web/ChatIframe.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+const chatChannel = interfaceConfig.CHAT_CHANNEL
+const chatToken = interfaceConfig.CHAT_TOKEN
+console.log("chatChannel",chatChannel)
```
Comment (SukantGujar) [inline react/features/chat/components/web/ChatIframe.js]: Remove

File: react/features/chat/components/web/ChatIframe.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+
+export default () => {
+    function authenticateIFrame() {
+      console.log("chatChannel",chatChannel)
```
Comment (SukantGujar) [inline react/features/chat/components/web/ChatIframe.js]: Remove

File: react/features/chat/components/web/ChatIframe.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+        setTimeout(() => {
+          document.getElementById('rcChannel').contentWindow.postMessage(
+            {
+              externalCommand: 'login-with-token',
+              // token: 'QQhWApDFcQ0qqZQZ73hww1mxXwWWqZx33i7In5dz2hs',
```
Comment (SukantGujar) [inline react/features/chat/components/web/ChatIframe.js]: Remove

**Review by SukantGujar — CHANGES_REQUESTED** [review]

File: react/features/chat/components/web/Chat.js (hunk 1 above)
Comment (rishi9684) [reply react/features/chat/components/web/Chat.js]: removed

File: react/features/chat/components/web/Chat.js (hunk 2 above)
Comment (rishi9684) [reply react/features/chat/components/web/Chat.js]: added CHAT_CHANNEL

File: react/features/chat/components/web/ChatIframe.js (hunk 4 above)
Comment (rishi9684) [reply react/features/chat/components/web/ChatIframe.js]: removed

File: react/features/chat/components/web/ChatIframe.js (hunk 5 above)
Comment (rishi9684) [reply react/features/chat/components/web/ChatIframe.js]: removed

**Review by SukantGujar — APPROVED** [review]

**Pull request merged**